pyclient/pydeephaven/table.py

```diff
@@ -0,0 +1,682 @@
+#
+#  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+
+from __future__ import annotations
+
+from typing import List
+
+import pyarrow
+
+from pydeephaven._combo_aggs import ComboAggregation
+from pydeephaven.constants import MatchRule, AggType, SortDirection
+from pydeephaven.dherror import DHError
+
+
+class Table:
+    """ A Table object represents a reference to a in-memory table on the server. It is the core data structure of
```

Comment:
An `in-memory` table is a specific type of Table on the server - I would drop "in-memory" here.

Reply:
will do.